brianlong/www.validators.app
# 186681424 - switch staker with withdrawer

#### What's this PR do?
- switching order of staker and withdrawer at stake explorer

#### How should this be manually tested?
- verify withdrawer and staker order links on stake-explorer page
- fill in above attributes within the form and search for results

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186681424)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
